ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/PartyAllocationLedgerEntry.scala

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.platform.sandbox.stores.ledger
+
+import java.time.Instant
+
+import com.daml.ledger.participant.state.v1.{ParticipantId, SubmissionId}
+import com.digitalasset.ledger.api.domain.PartyDetails
+
+//TODO BH consider removing duplication of this class
+sealed abstract class PartyAllocationLedgerEntry() extends Product with Serializable {
+  val submissionId: String
+  val participantId: ParticipantId
+  val recordTime: Instant
+  def value: String
```

Comment:
Same comment as for `PackageUploadLedgerEntry`. Please follow the `ConfigurationEntry` pattern.